sdk/network/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/PublicIPSkuType.java

```diff
@@ -19,6 +19,9 @@ public final class PublicIPSkuType {
     /** Static value Standard for PublicIPSkuType. */
     public static final PublicIPSkuType STANDARD = new PublicIPSkuType(PublicIpAddressSkuName.STANDARD);
 
+    /** Static value StandardV2 for PublicIPSkuType. */
+    public static final PublicIPSkuType STANDARD_V2 = new PublicIPSkuType(PublicIpAddressSkuName.STANDARD_V2);
```

Comment:
there is a V2 in service https://github.com/Azure/azure-sdk-for-java/blob/0321cfb05cfd830eb99c17f225e87dd6797abe73/sdk/network/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/PublicIpAddressSkuName.java#L27